Common/Data/UniverseSelection/UserDefinedUniverse.cs

```diff
@@ -322,5 +317,37 @@ protected virtual void OnCollectionChanged(NotifyCollectionChangedEventArgs e)
                                                                    startTimeUtc: currentTimeUtc,
                                                                    endTimeUtc: maximumEndTimeUtc));
         }
+
+        /// <summary>
+        /// Tries to remove the specified security from the universe.
+        /// </summary>
+        /// <param name="utcTime">The current utc time</param>
+        /// <param name="security">The security to be removed</param>
+        /// <returns>True if the security was successfully removed, false if
+        /// we're not allowed to remove or if the security didn't exist</returns>
+        internal override bool RemoveMember(DateTime utcTime, Security security)
+        {
+            if (base.RemoveMember(utcTime, security))
+            {
+                RemoveAndKeepTrack(security.Symbol);
+                return true;
+            }
+            return false;
+        }
+
+        private bool RemoveAndKeepTrack(Symbol symbol)
+        {
+            var toBeRemoved = _subscriptionDataConfigs.Where(x => x.Symbol == symbol).ToList();
+            var removedSymbol = _symbols.Remove(symbol);
+
+            if (toBeRemoved.Any() || removedSymbol)
```

Comment:
could save a couple cycles here by doing:
``` csharp
if (removedSymbol || toBeRemoved.Any())
{
    //
}
```
This will cause us to only evaluate the LINQ expression _if_ `removedSymbols == false`

Reply:
Good catch 👍